memote/support/consistency.py

```diff
@@ -479,14 +484,17 @@ def find_disconnected(model):
     return [met for met in model.metabolites if len(met.reactions) == 0]
 
 
-def find_metabolites_produced_with_closed_bounds(model):
+def find_metabolites_produced_with_closed_bounds(model, threshold=1E-07):
```

Comment:
Do you expect the thresholds to be different/specific for each test? Should we perhaps set it in a global variable?